e2e-tests/playwright/utils/UIhelper.ts

```diff
@@ -90,7 +90,17 @@ export class UIhelper {
   }
 
   async openSidebar(navBarText: string) {
-    await this.page.locator(`nav a`).filter({ hasText: navBarText }).click();
+    await this.page.click(`nav a:has-text("${navBarText}")`);
+  }
+
+  async verifyTableHeaders(expectedHeaders: string[]) {
+    for (const header of expectedHeaders) {
+      const headerLocator = this.page.locator(
+        `${UIhelperPO.MuiTable} thead th >> text="${header}"`,
+      );
+      const count = await headerLocator.count();
+      expect(count).toBeGreaterThan(0);
```

Comment:
Could we use toBeVisible instead? I believe it would accomplish the same thing while also ensuring visibility.